src/main/java/com/homeworks/homework20/controller/AppController.java

```diff
@@ -0,0 +1,58 @@
+package com.homeworks.homework20.controller;
+
+import com.homeworks.homework20.model.User;
+import com.homeworks.homework20.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+
+import javax.validation.Valid;
+import java.util.List;
+import java.util.Map;
+
+@Controller
+@ComponentScan(basePackages = "com.homeworks")
+public class AppController {
+    @Autowired
+    private UserService userService;
```

Comment:
It is better to use a constructor for dependency injection.
https://www.vojtechruzicka.com/field-dependency-injection-considered-harmful/

Reply:
You haven't answered yet. Please provide an answer.